jwst/datamodels/schemas/pathloss.schema.yaml

```diff
@@ -0,0 +1,91 @@
+title: Pathloss correction data model
+allOf:
+- $ref: core.schema.yaml
+- $ref: referencefile.schema.yaml
+- type: object
+  properties:
+    pointsource:
+      type: object
+      properties:
+        data:
+          title: Point source pathloss
+          fits_hdu: ps
+          default: 1.0
+          ndim: 3
+          datatype: float32
+        err:
+          title: Point source pathloss variance
+          fits_hdu: psvar
+          default: 0.0
+          ndim: 3
+          datatype: float32
+        wcs:
```

Comment:
Is this WCS info that's contained within the reference file, from which the wavelengths of the reference data are computed later? The WCS of the science data is of course already available in the meta.wcs object.